test/data/recordings/TestAccAttackProtectionBreachedPasswordDetection.yaml

```diff
@@ -19,7 +19,7 @@ interactions:
         Content-Type:
           - application/json
         User-Agent:
-          - Go-Auth0-SDK/0.12.0
+          - Go-Auth0-SDK/0.15.0
```

Comment:
Updating the recording with AUTH0_HTTP_RECORDINGS=on. I'll make sure to update the contributing section as well (probably next week) for terraform like we did for go-auth0 to make it easier and have some handy make commands to regenerate automatically just for 1 test @nialdaly